src/components/ProgressBar.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useRef } from "react";
+import * as d3 from "d3";
+import "./ProgressBar.css";
+
+const CheckColor = (value) => {
+  if (value < 75) return "red";
+  if (value > 90) return "green";
+  return "darkViolet";
+};
```

Comment:
functions should be camel-cased. `checkColor` instead of `CheckColor`

Reply:
Got it. will make the changes from here on.